docs/README.md

```diff
@@ -18,14 +18,19 @@ The documentation uses [GitHub issues](https://github.com/Sylius/Sylius/issues).
 Build
 -----
 
-If you contribute to documentation you will want to check how it looks after your changes.  
-To be able to build the documentation, install [Sphinx](http://sphinx-doc.org/).
+To test the documentation before a commit:
 
-1. [Install `pip`, Python package manager](https://pip.pypa.io/en/stable/installing/)
-2. Download documentation requirements: `$ pip install -r requirements.txt`
-3. Build the documentation:`$ sphinx-build -b html . build`
+* Install [Python](https://www.python.org/)
 
-Documentation index is `build/index.html`.
+    `$ brew install python`
+
+* Install [Sphinx](http://www.sphinx-doc.org/en/stable/)
+
+    `$ sudo pip install Sphinx`
```

Comment:
Anyway, we should not use `sudo` with `pip`: http://stackoverflow.com/questions/21055859/what-are-the-risks-of-running-sudo-pip